src/main/java/com/hermesworld/ais/galapagos/applications/ApplicationsService.java

```diff
@@ -40,6 +41,9 @@ default List<ApplicationMetadata> getAllApplicationMetadata(String environmentId
     CompletableFuture<ApplicationMetadata> registerApplicationOnEnvironment(String environmentId, String applicationId,
             JSONObject registerParams, OutputStream outputStreamForSecret);
 
+    CompletableFuture<Integer> getServiceAccountIdForApplication(String applicationId, String environmentId)
```

Comment:
This method should not be in ApplicationsService, as it is completely Confluent-specific.